.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -31,7 +31,7 @@ phases:
   - phase: build_vhd
     queue:
       name: Hosted Ubuntu 1604
-      timeoutInMinutes: 120
+      timeoutInMinutes: 360
```

Comment:
hit a lot of timeouts with 120 min limit lately...

Reply:
do we know why? we have another file with the same setting that should be in sync with this one 
.vsts-vhd-builder-release.yaml

Reply:
I actually already increase the other one to 3 hours because I hit the same problem for 2021.01.06 VHD release: https://github.com/Azure/AgentBaker/commit/734790000e3250f70c4433725bbdfde626508c35

my math needs some work -__- meant to put 180 here instead of 360 (3h not 6h), but really we just want to avoid a timeout anyway.

Reply:
most of the builds still finish in 120m, but often one or two will take a bit more than 2 hours, failing the process immediately at the 2h mark.

Reply:
got you thanks.